### DISCUSSION TITLE
How to use add_ColorValuesChanged?

### DISCUSSION THREAD

**Discussion by Ferry-200** [halildurmus/dartwinrt#387] [Q&A]
I want to change my application's theme when Windows changing it's theme. 

In C++, we can pass a function to `ColorValuesChanged` and it will be executed when theme changing. 

But how can I do that in Dart?

**Comment by halildurmus** [comment]
Callbacks are not supported yet (#335).

**Reply by Ferry-200** [reply]
Well. Maybe I have to turn to rust's windows crate and the `flutter rust bridge`. Anyway, thanks for telling me.
